go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/prometheus/client_model
+
+go 1.9
+
+require (
+	github.com/golang/protobuf v1.2.0
+	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4 // indirect
```

Comment:
For sanity, I would pin it to the same version that we use in the other Prometheus repositories which is `v0.0.0-20181108010431-42b317875d0f`.